app/assets/images/payment_icons/ratepay.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img" aria-labelledby="pi-ratepay"><title id="pi-ratepay">RatePay</title><g fill-rule="evenodd" clip-rule="evenodd" id="pi-ratepay-background_ratepay"><path fill="#FFF" d="M36 24H2c-1.333 0-2-.7-2-2V2C0 .667.7 0 2 0h34c1.333 0 2 .7 2 2v20c0 1.333-.667 2-2 2z"/><path fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="3" stroke-opacity=".07" d="M36 24c1.333 0 2-.7 2-2V2c0-1.333-.7-2-2-2H2C.7 0 0 .7 0 2v20c0 1.333.7 2 2 2h34z"/></g><path fill-rule="evenodd" clip-rule="evenodd" fill="#E4003E" d="M14.2 5.3c1.3-.833 2.75-1.267 4.35-1.3 1.467 0 2.851.367 4.15 1.1l-7.35 7.55L11.4 8.8a8.01 8.01 0 012.8-3.5zm11.7 3.5c.466.967.699 2 .699 3.1 0 2.234-.75 4.133-2.25 5.7-1.566 1.534-3.449 2.334-5.649 2.4-1.066 0-2.116-.2-3.15-.6L25.9 8.8z" id="pi-ratepay-icon_ratepay"/></svg>
```

Comment:
Same comments applied here: unused `id` attributes and border needs to follow other icons.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img" aria-labelledby="pi-ratepay"><title id="pi-ratepay">RatePay</title><g fill-rule="evenodd" clip-rule="evenodd"><path fill="#FFF" d="M36 24H2c-1.333 0-2-.7-2-2V2C0 .667.7 0 2 0h34c1.333 0 2 .7 2 2v20c0 1.333-.667 2-2 2z"/><path fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="3" stroke-opacity=".07" d="M36 24c1.333 0 2-.7 2-2V2c0-1.333-.7-2-2-2H2C.7 0 0 .7 0 2v20c0 1.333.7 2 2 2h34z"/></g><path fill-rule="evenodd" clip-rule="evenodd" fill="#E4003E" d="M14.2 5.3c1.3-.833 2.75-1.267 4.35-1.3 1.467 0 2.851.367 4.15 1.1l-7.35 7.55L11.4 8.8a8.01 8.01 0 012.8-3.5zm11.7 3.5c.466.967.699 2 .699 3.1 0 2.234-.75 4.133-2.25 5.7-1.566 1.534-3.449 2.334-5.649 2.4-1.066 0-2.116-.2-3.15-.6L25.9 8.8z"/></svg>
```